presto/dbapi.py

```diff
@@ -281,12 +281,24 @@ def _prepare_statement(self, operation, statement_name):
 
         raise presto.exceptions.FailedToObtainAddedPrepareHeader
 
+    def _format_prepared_param(self, param):
```

Comment:
please move this below _get_added_prepare_statement_presto_query

Reply:
moved